client/packages/programs/src/JsonForms/components/IdGenerator.tsx

```diff
@@ -268,6 +281,31 @@ const useUniqueProgramEnrolmentIdValidation = () => {
   };
 };
 
+const useUniqueProgramPatientCodeValidation = () => {
+  const { mutateAsync: fetchPatientCodes } =
+    usePatient.document.usePatientsPromise();
+
+  // returns error if validation fails (patient code already in use)
+  return async (
+    code: string,
+    documentName: string
```

Comment:
The documentName doesn't make sense here, the patient doesn't has a document like the program enrolment has... Could you please remove this parameter and the validation?